sdk/identity/azcertificatemanager/autorest.md

```diff
@@ -0,0 +1,30 @@
+## Go
+
+``` yaml
+
+clear-output-folder: false
+export-clients: true
+go: true
+input-file: C:\Users\arpitsaxena\source\repos\azure-rest-api-specs-pr\specification\pki\Microsoft.Pki\preview\2022-09-01-preview\pki.json
```

Comment:
https://github.com/Azure/azure-rest-api-specs-pr/blob/a9b63df78c701d686e50c0e1a1d457c1a0b4b114/specification/pki/Microsoft.Pki/preview/2022-09-01-preview/pki.json or whatever is appropriate.